apps/qubit/modules/settings/actions/globalAction.class.php

```diff
@@ -273,6 +275,16 @@ protected function updateGlobalSettings()
       $setting->save();
     }
 
+    // Escape queries, add setting if it's not already created (to avoid adding it in a migration)
+    if (null === $setting = QubitSetting::getByName('escape_queries'))
+    {
+      $setting = QubitSetting::createNewSetting('escape_queries', null);
```

Comment:
Do you think it'd make sense to make the default here the set of special characters that make ES produce an error instead of null?